samples/host_exerciser/host_exerciser_cmd.h

```diff
@@ -111,6 +111,11 @@ class host_exerciser_cmd : public test_command
               he_lpbk_cfg_.TputInterleave = host_exe_->he_interleave_;
         }
 
+        // Set Interrupt test mode
+        if (host_exe_->he_interrupt_ > 0) {
```

Comment:
The parameter is declared to accept a Range(0, 3).  Doesn't this mean that 0 is a valid interrupt index?

Reply:
interrupt index has to fixed during integration. we don't know interrupt number